src/udp_wrap.cc

```diff
@@ -139,7 +139,7 @@ void UDPWrap::New(const FunctionCallbackInfo<Value>& args) {
 
 
 void UDPWrap::GetFD(Local<String>, const PropertyCallbackInfo<Value>& args) {
-  int fd = -1;
+  int fd = UV_EINVAL;
```

Comment:
`UV_EBADF` seems reasonable.  I probably said `UV_EINVAL` as a first suggestion, not something final.